share/functions/help.fish

```diff
@@ -133,13 +133,15 @@ function help --description 'Show help for the fish shell'
     end
 
     set -l page_url
-    if test -f $__fish_help_dir/index.html
-        # Help is installed, use it
+    if test -f $__fish_help_dir/(string match -r '^.+html' $fish_help_page)
+        # Help page is available locally, use it
```

Comment:
I wonder if this change is necessary, it seems it would cover up broken installations

Reply:
Yeah, shall I revert it?

If `index.html` exists then the rest of the docs should be there too, assuming the files haven't been deleted or anything.

Reply:
Is help ever not installed locally?

Reply:
Just built from master following the steps from the README to test this PR, and the docs were not installed:
```
> ll $__fish_help_dir
total 128K
-rw-r--r-- 1 root root 77K Nov 28 14:05 CHANGELOG.md
```

Reply:
You need Sphinx installed to build the docs. If they aren't installed, the `help` command should use the online version.